tools/find_inverse.cpp

```diff
@@ -127,6 +131,50 @@ bool s_method_1(int den, int64_t mul, int sh_post, int bits) {
     return true;
 }
 
+bool srz_method_0(int den, int sh_post, int bits) {
+    int64_t min = -(1L << (bits - 1)), max = (1L << (bits - 1)) - 1;
+    for (int64_t num = min; num <= max; num++) {
+        // for (int iter = 0; iter < 1000000L; iter++) {
```

Comment:
Why is this commented out? If it's being left in for (eg) debugging purposes, please say so.

Reply:
Fixed (deleted)